x-pack/test/cases_api_integration/common/lib/attachments.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type SuperTest from 'supertest';
+import { CASES_INTERNAL_URL } from '@kbn/cases-plugin/common/constants';
+import { BulkGetCommentsResponse } from '@kbn/cases-plugin/common/api';
+import { User } from './authentication/types';
+import { superUser } from './authentication/users';
+import { getSpaceUrlPrefix } from './utils';
+
+export const bulkGetAttachments = async ({
+  supertest,
+  attachmentIds,
+  caseId,
+  expectedHttpCode = 200,
+  auth = { user: superUser, space: null },
+}: {
+  supertest: SuperTest.SuperTest<SuperTest.Test>;
+  attachmentIds: string[];
```

Comment:
`BulkGetArgs`(and `BulkGetCommentsRequestRt`) defines the `attachmentIDs` as being `string[] | string` but in the tests, we only use ever `string[]`.

It would be nice to ensure things also work when just a string is sent in the body.